R-package/NEWS.md

```diff
@@ -1,3 +1,8 @@
+## RGF 1.0.7
+* We've modified the *package.R* file so that messages are printed to the console whenever Python or any of the required modules is not available. Moreover, for the R-package testing the conda environment parameter is adjusted ( this applies to the RGF-team Github repository and not to the CRAN package directly )
```

Comment:
Need to be updated?

Reply:
`Github` -> `GitHub`

Reply:
Yes you're right. If you don't mind I'll fixed it before I submit the next version to CRAN.

Reply:
Sure, do it please.